oauth2_http/java/com/google/auth/oauth2/OAuth2Credentials.java

```diff
@@ -254,7 +254,13 @@ private AsyncRefreshResult getOrCreateRefreshTask() {
               new Callable<OAuthValue>() {
                 @Override
                 public OAuthValue call() throws Exception {
-                  return OAuthValue.create(refreshAccessToken(), getAdditionalHeaders());
+                  OAuthValue result = OAuthValue.create(refreshAccessToken(), getAdditionalHeaders());
+
```

Comment:
Would it be better to move `finishRefreshAsync` up onto this line?

Reply:
Yes, that might be better.

I don't know if there is any listener that relies on the task being completed, so I didn't want to change it.